openapi/mojaloop-connector.yaml

```diff
@@ -0,0 +1,281 @@
+openapi: 3.0.3
+info:
+  title: Rafiki Mojaloop Connector
+  description: |-
+    The Rafiki Mojaloop Connector is a REST API with 2 resource types: **quote** and **transfer**.
+
+    All resource and collection resource representations use JSON and the media-type `application/json`.
+
+    Access to resources and permission to execute the methods exposed by the API is determined by the grants given to the client represented by an access token used in API requests.
+
+  termsOfService: https://interledger.org/
+  contact:
+    email: tech@interledger.org
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: '1.0'
+externalDocs:
+  description: Find out more about Rafiki
+  url: https://rafiki.dev/
+servers:
+  - url: https://rafiki.connector.mojaloop/
+tags:
+  - name: quote
+    description: quote related operations
+  - name: transfer
+    description: transfer related operations
+paths:
+  /mjl-quote:
+    post:
+      tags:
+        - quote
+      summary: Request a new Mojaloop quote.
+      description: Request a new Mojaloop quote, which will be routed to the CNP (Cross Network Provider) Mojaloop Hub instance.
+      operationId: requestQuote
+      requestBody:
+        description: Create a new quote request.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/mjl-quote-req'
+        required: true
+      parameters:
+        - name: fspiop-source
+          in: header
+          description: 'The DFSP that requested the quote'
+          required: false
+          schema:
+            type: string
+        - name: fspiop-destination
+          in: header
+          description: 'The DFSP destination'
+          required: false
+          schema:
+            type: string
```

Comment:
What is the difference between these and the payer/payee `fspId`s?

Reply:
Since the `fspId` is optional in the `mjl-party` payload, the header is not. from what I can gather from the FSIOP API is that the header info is utilized to create `fspiopOpaqueState` values.
I would not have designed it like this. 🤷‍♂️

Reply:
It looks like it is required here, no?
https://github.com/interledger/rafiki/pull/1948/files#diff-9bfa74037f55ce65ba743bd62e2e0e52d0146b5b56b244f72b46fbffc0e14bd7R233-R237